apps/docs/README.md

```diff
@@ -6,24 +6,28 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 
 - stylex actual package, and then the @stylexjs/babel-plugin plugin in node_modules
 
+```bash
+$ npm run build -w @stylexjs/stylex -w @stylexjs/shared -w @stylexjs/eslint-plugin -w @stylexjs/babel-plugin
```

Comment:
`npm run build --workspaces` is all that's needed

Reply:
That builds all the packages/apps in the workspace, the documentation above only mentions the stylex packages and eslint plugin, so I only added builds that are required for docs to run (minimizes build time for first time contributors looking to only contribute to docs as size of monorepo increases). I could change it to build the entire workspace if that is the desired behavior 😀